pkg/k8shandler/index_management.go

```diff
@@ -41,10 +41,29 @@ func (elasticsearchRequest *ElasticsearchRequest) CreateOrUpdateIndexManagement(
 			return err
 		}
 	}
+	if err := indexmanagement.ReconcileCurationConfigmap(elasticsearchRequest.client, elasticsearchRequest.cluster); err != nil {
+		return err
+	}
+	primaryShards := getDataCount(elasticsearchRequest.cluster)
+	for _, mapping := range spec.Mappings {
+		policy := policies[mapping.PolicyRef]
+		if err := indexmanagement.ReconcileRolloverCronjob(elasticsearchRequest.client, elasticsearchRequest.cluster, policy, mapping, primaryShards); err != nil {
+			result = err
+			logger.Errorf("There was an error reconciling the rollover cronjob for policy %q", policy.Name)
+		}
+		if err := indexmanagement.ReconcileCurationCronjob(elasticsearchRequest.client, elasticsearchRequest.cluster, policy, mapping, primaryShards); err != nil {
+			result = err
```

Comment:
its possible for this to override the prior `err` from the `RolloverCronJob` call...